### DISCUSSION TITLE
Bank recommendations

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#1228] [General]
Please recommend a reliable bank

**Comment by hernessteve** [comment]
It really depends on your specific purposes. What do you need a new bank for? Are you looking for better interest rates on savings, lower fees for everyday banking, or perhaps more convenient online services? Maybe you’re interested in features like budgeting tools, investment options, or rewards programs.

**Comment by MirkosMangle** [comment]
Identifying your priorities can help narrow down your choices and find a bank that aligns with your financial goals. For instance, if you're planning to save for a major purchase, a bank with high-interest savings accounts might be ideal. On the other hand, if you frequently travel, you might want a bank that offers no foreign transaction fees. [broadway bank customer service](https://www.pissedconsumer.com/company/broadway-bank/customer-service.html) seems to be reliable from my experience
